build.gradle

```diff
@@ -44,6 +44,14 @@ def envVersion = System.getenv('VERSION')
 version = envVersion == null ? 'indev' : envVersion
 java.toolchain.languageVersion = JavaLanguageVersion.of(17)
 
+
+sourceSets {
+    test
+    main {
+        runtimeClasspath += sourceSets.test.output
+    }
```

Comment:
I'm very concerned by this, does this not dump the entire test outputs onto the class path in non test environments? i.e. `runClient` will have test sources present (no bueno)